en/docs/install-and-setup/upgrading-wso2-api-manager/what-has-changed.md

```diff
@@ -22,6 +22,11 @@ With these new features, we have removed the API level mediation policies and if
 
 Previous mediation related resources from the Publisher and Admin REST API have been removed and you can achieve the same functionality via the operation-policies resource in the Publisher REST API.
 
+Mediation policies that you have attached to API types other than REST, i.e., SOAP, GraphQL, etc., will no longer be manageable after you have migrated to 4.1.0. With the API Policies feature, the policy support was only provided for REST APIs and extending the same for other API types is still in the roadmap, and we hope to send a WUM update to 4.1.0 to add this support.
+
+!!! note
+    Even though the functionality is still intact (runtime is not affected), the design time is affected as you can no longer manage policies using the UI, or the file system. To further elaborate on this, if you had policies attached to an API and you've migrated to 4.1.0, those policies are still attached to the said API, but they are immutable.
```

Comment:
At the start of the note, shall we mention that we are talking about API types other than REST APIs? I feel like its not clear enough that we're talking about SOAP, GraphQL, etc. here.

Reply:
Reworded this. Please check now.

Reply:
LGTM